controllers/script.js

```diff
@@ -246,11 +246,14 @@ var getScriptPageTasks = function (aOptions) {
 
   // Setup the flagging UI
   tasks.push(function (aCallback) {
-    // Can't flag when not logged in or when user owns the script.
-    if (!authedUser || aOptions.isOwner) {
-      aCallback();
-      return;
-    }
+    // WARNING: Possible duplicated code from flagLib.flaggable
+
+      // Can't flag when not logged in or when user owns the script.
+      if (!authedUser || aOptions.isOwner) {
```

Comment:
**NOTES**
* Same tests here as [...blob/`703df10`/libs/flag.js#L22](https://github.com/OpenUserJs/OpenUserJS.org/blob/703df104da6b3667687da8311c7c846d76bf9fc9/libs/flag.js#L22) and [.../blob/`703df10`/libs/flag.js#L32](https://github.com/OpenUserJs/OpenUserJS.org/blob/703df104da6b3667687da8311c7c846d76bf9fc9/libs/flag.js#L32). Looks okay to remove this entire `WARNING` block since the early test is really "dumbed down" compared to the `flaggable` routine.